app/lib/parse.ts

```diff
@@ -48,3 +48,22 @@ export const parseClueReferences = (text: string): ClueReferenceData[] => {
   }
   return refs;
 };
+
+export function parseClueEnumeration(str: string): string | null {
```

Comment:
Not entirely satisfied with this -- open to better approaches here!

Reply:
I think it seems OK - the only thing I'm curious about is when an enum would ever have  a depth > 1? It almost feels like if there is a nested paren then the constructor is doing something more "clever" than an enumeration and we shouldn't count it as one. But if there are examples otherwise then I'm all ears - I'm not too familiar w/ the different enum formats.

Reply:
Fair! I had been looking at this wiki https://www.puzzles.wiki/wiki/Enumeration which lists this example:
> Stay (I Missed You) --> (4 (1 6 3)) or (4 1 6 3)

I guess the implication being that, in addition to numbers and information like `abbr.`, enumerations can include any punctuation in the answer phrase (e.g. `-`, `,`, `'`, etc.)? So for detection, I tried to go for something like: "if the clue ends in a matching pair of parentheses and their contents include a digit, that's an enumeration".

Maybe a stricter definition makes more sense though! What do you think?

Reply:
That makes sense! Let's keep it as is.